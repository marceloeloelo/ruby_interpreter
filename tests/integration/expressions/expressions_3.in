(1 + 1)
@var = (1 + 1)
@var_1 = (@var_2 = (@var_3 = 1 + 2))
(1 + 2) * (3 + 4)
("test")
