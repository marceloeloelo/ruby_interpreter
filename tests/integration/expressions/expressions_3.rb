(1 + 1)
var = (1 + 1)
puts (var)

var_1 = (var_2 = (var_3 = 1 + 2))
puts (var_1)
puts (var_2)
puts (var_3)

(1 + 2) * (3 + 4)
puts((1 + 2) * (3 + 4))

("test")
puts(("test"))
