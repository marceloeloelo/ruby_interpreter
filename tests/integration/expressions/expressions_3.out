2
3
3
3
21
test
